src/test/java/de/tum/in/www1/artemis/SubmissionExportIntegrationTest.java

```diff
@@ -220,6 +220,15 @@ public void testExportAll_IOException() throws Exception {
         mockedFiles.close();
     }
 
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void testExportTextExercise_IOException() throws Exception {
+        MockedStatic<Files> mockedFiles = mockStatic(Files.class);
+        mockedFiles.when(() -> Files.newOutputStream(any(), any())).thenThrow(IOException.class);
```

Comment:
maybe I am missing something... or misunderstanding the concept of the argument matchers in static mocks ❓  But as far as I can see, you are creating the mock in order to throw the IO Exception when calling `Files.newOutputStream(zipFilePath)` in `createZipFile` method of the `ZipFileService` : https://github.com/ls1intum/Artemis/blob/9eb9e03f816940b426c5649b1c85af63490312fb/src/main/java/de/tum/in/www1/artemis/service/ZipFileService.java#L46 -> I debugged the test and ran into this line directly before triggering the catch statement in the `TextExerciseResource` that you wanted to cover. However, this method takes one input (`zipFilePath`), why do you have to two `any()` captures here? I also tested your code with one and the test still passes. I am confused 😄

Reply:
I do not have really good knowledge about MockStatic. I just try to apply other test case (`testExportAll_IOException()`) for my case. I did not try the case you mentioned, maybe we could ask this to someone else.

Reply:
ah I guess it's due to `public static OutputStream newOutputStream(Path path, OpenOption... options)` having varargs (0-many options), so only path is required. That should be the reason why it makes no difference in our case, if we use `Files.newOutputStream(any(), any())` or `Files.newOutputStream(any())` as our code calls the method only with one argument.

Reply:
hmm, I see. really appreciate it for your notice. 😉  so, it does not have big impact, do you think that I should pass only one argument?

Reply:
as it is done the same in the newly written [documentation](https://github.com/ls1intum/Artemis/pull/3297) as well with the two any()s, I would say, just leave it like this ;) it can be resolved!